.circleci/config.yml

```diff
@@ -81,7 +81,7 @@ jobs:
         enum: [ development, staging, production ]
     environment:
       SAM_CONFIG_FILE: samconfig.toml
-      SAM_LAMBDA_CONFIG_ENV: <<parameters.dc-environment>>
+      DC_ENVIRONMENT: <<parameters.dc-environment>>
       SAM_PUBLIC_CONFIG_ENV: <<parameters.dc-environment>>-public-access
```

Comment:
Similar to the website, I think we can also remove the `SAM_PUBLIC_CONFIG_ENV` var. I don't think it is used anywhere.